validation and qualification/README.md

```diff
@@ -1,16 +1,25 @@
-* [OSPSuite.SimModel](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-simmodel/builds/24100473/tests)
-* [OSPSuite.SimModel.Solver.CVODES](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-simmodel-solver-cvodes/builds/24073518/tests)
-* [OSPSuite.FuncParser](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-funcparser/builds/22310440/tests)
-* [OSPSuite.DataBinding.DevExpress](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-databinding-devexpress/builds/23174184/tests)
-* [OSPSuite.DataBinding](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-databinding/builds/22913138/tests)
-* [OSPSuite.Utility](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-utility/builds/22701760/tests)
-* [OSPSuite.TeXReporting](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-texreporting/branch/master/tests)
+* [PK-Sim](https://ci.appveyor.com/project/open-systems-pharmacology-ci/pk-sim/builds/33493875/tests)
+* [MoBi](https://ci.appveyor.com/project/open-systems-pharmacology-ci/mobi/builds/33493863/tests)
+* [Installation Validator](https://ci.appveyor.com/project/open-systems-pharmacology-ci/installationvalidator/builds/33493856/tests)
+* [OSPSuite.Core](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-core/builds/33492251/tests)
+* OSPSuite.SimModel
+  * [Windows](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-simmodel/builds/31428566/job/u044r6id42p0jwmg/tests)
+  * [Linux (Ubuntu 18.04)](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-simmodel/builds/31428566/job/wyjatqic8i58g2cn/tests)
+* [OSPSuite.SimModel.Solver.CVODES](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-simmodel-solver-cvodes/builds/31394354/job/sijwuswr2abe3yk4/tests)
+* OSPSuite.FuncParser
+  * [Windows](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-funcparser/builds/31401683/job/g74hurc205p70uo7/tests)
+  * [Linux (Ubuntu 18.04)](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-funcparser/builds/31401683/job/0u8h2vsjth3kr94h/tests)
+* [OSPSuite.DataBinding.DevExpress](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-databinding-devexpress/builds/29420584/tests)
+* [OSPSuite.DataBinding](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-databinding/builds/29419442/tests)
+* [OSPSuite.Utility](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-utility/builds/29194230/tests)
+* [OSPSuite.TeXReporting](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-texreporting/builds/28672332/tests)
+* [OSPSuite.Serializer](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-serializer/builds/25501936/tests)
 
 ### [Batch comparison tests](BatchComparison)
 
```

Comment:
Overview of all Qualification/Evaluation Reports, Ranorex Tests, AppVeyor-Tests made for the Release